### PR TITLE
dvl beta integration doc

### PR DESCRIPTION
**Pull request by jaxxzer**

### CODE REVIEW

File: developers/dvl-integration.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+The DVL integration with ArduSub is in a BETA stage and is currently unsupported for normal use. This document is provided to instruct beta testers and developers how to test the BETA stage integration. There is no guarantee that the DVL integration will work for any particular application.
+
+## Supported DVLs
+
+The [Waterlinked A50 DVL](https://waterlinked.com/product/dvl-a50/) is the only supported DVL at the moment in the beta integration.
+
+### A50 DVL configuration
+
+Set it up to use the [Static IP address](https://waterlinked.github.io/docs/dvl/dvl-a50-details/#network-config) 192.168.2.94
```
Comment (jaxxzer) [inline developers/dvl-integration.md:11]: Does this conflict with WL UGPS?

The DVL should be configured with a static ip address of 192.168.2.94. Follow Water Linked's instructions here.

Comment (Williangalvani) [reply developers/dvl-integration.md:11]: YES! that should be .95, my mistake.

File: developers/dvl-integration.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+The DVL integration with ArduSub is in a BETA stage and is currently unsupported for normal use. This document is provided to instruct beta testers and developers how to test the BETA stage integration. There is no guarantee that the DVL integration will work for any particular application.
+
+## Supported DVLs
+
+The [Waterlinked A50 DVL](https://waterlinked.com/product/dvl-a50/) is the only supported DVL at the moment in the beta integration.
+
+### A50 DVL configuration
+
+Set it up to use the [Static IP address](https://waterlinked.github.io/docs/dvl/dvl-a50-details/#network-config) 192.168.2.95
```
Comment (jaxxzer) [inline developers/dvl-integration.md:11]:
```suggestion
The DVL must be configured with a static IP address of 192.168.2.95. Follow Water Linked's instructions here.
```

File: developers/dvl-integration.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+
+**Restart DVL Service:**
+
+Restarts the DVL service.
+
+
+**Set New Origin:**
+
+Used for dead reckoning, This makes the ROV show up in QGC at the selected location. This is likely to be not as good as the DVL's own dead reckoning.
```
Comment (jaxxzer) [inline developers/dvl-integration.md:61]:
```suggestion
Used for dead reckoning; this makes the ROV show up in QGC at the selected location. The position displayed in QGC is calculated by the autopilot independently of the position showed in the Water Linked DVL web interface, which is calculated by the DVL itself.

File: developers/dvl-integration.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+**Status:**
+
+Shows the current Driver status, useful for troubleshooting.
+
+
+## ArduSub Firmware
+
+The DVL data is sent by the VISUAL_ODOMETRY mavlink message. This message is supported in ArduSub 4.1-beta and onward. The ArduSub 4.1-beta firmware is recommended for testing with DVL.
+The Firmware **must** be downloaded from the ardupilot servers [here](https://firmware.ardupilot.org/Sub/beta/Pixhawk1/ardusub.apj)
```
Comment (jaxxzer) [inline developers/dvl-integration.md:72]:
```suggestion

```

**Review by Williangalvani — APPROVED** [review]

**Pull request merged**